src/core/css/SearchResult.scss

```diff
@@ -104,10 +104,6 @@
     object-fit: cover;
     object-position: top right;
     width: 100%;
-
-    [dir=rtl] & {
```

Comment:
This is removed since for RTL FF the header images are still anchored right so there's no need to change that when the site is being viewed in RTL.